### PR TITLE
Add DSL support for OpenShift Whereabouts CNI Model & `user.openshift.io/v1#Identitiy` to OpenShiftClient DSL

### PR DESCRIPTION
**Pull request by rohanKanojia**
## Description

Add DSL support for OpenShift Whereabouts CNI Model to OpenShiftClient DSL
    
+ Added new module `openshift-model-whereabouts/`
+ Add new DSL entrypoint `openShiftClient.whereabouts()` for accessing Whereabouts resources
      - IPPool: `openShiftClient.whereabouts().ippools()`
      - OverlappingRangeIPReservation: `openShiftClient.whereabouts().overlappingRangeIpReservations()`
+ Added DSL entrypoint `openShiftClient.identities()` for Identity
    resource in `user.openshift.io/v1` apiGroup

Related to #2949 

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [X] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [X] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [X] I Added [CHANGELOG](../CHANGELOG.md) entry regarding this change
 - [X] I have implemented unit tests to cover my changes
 - [X] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [ ] I tested my code in Kubernetes
 - [X] I tested my code in OpenShift

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: kubernetes-model-generator/openshift-model/src/generated/java/io/fabric8/openshift/api/model/Identity.java [hunk 1]
```diff
@@ -59,7 +58,7 @@
 @VelocityTransformations({
     @VelocityTransformation(value = "/manifest.vm", outputPath = "openshift.properties", gather = true)
 })
-public class Identity implements HasMetadata, Namespaced
```
Comment (manusa) [inline kubernetes-model-generator/openshift-model/src/generated/java/io/fabric8/openshift/api/model/Identity.java:62]: How come this one changed? I can't see any modifications in existing go code

Comment (rohanKanojia) [reply kubernetes-model-generator/openshift-model/src/generated/java/io/fabric8/openshift/api/model/Identity.java:62]: see `kubernetes-model-generator/pkg/schemagen/generate.go` . I've added it to Cluster Scoped resources list

Comment (manusa) [reply kubernetes-model-generator/openshift-model/src/generated/java/io/fabric8/openshift/api/model/Identity.java:62]: Nevermind, saw the change now, the GitHub file filter is not working well.

I assume this is correcting a previous misconfiguration.

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Pull request merged**